guides/pre-release-week.md

```diff
@@ -0,0 +1,68 @@
+# Pre-Release Week <!-- omit in toc -->
+
+> ### **What's the pre-release week?**
+>
+> One week before the official release of MeiliSearch, the Core team creates a release candidate (RC) version of MeiliSearch. For the integration team, the main purpose of this RC is to implement the new features (in the integrations tools) related to the next release by testing them against the RC version.
+>
+> So **the pre-release week is a sprint for the Integration team** who needs to complete all the tests and integrate changes before the official release of MeiliSearch.
```

Comment:
Maybe would be interesting to comment that not EVERY change will be integrated immediately. Because this seems to be the case often?